### DISCUSSION TITLE
Can I use components inside my content articles??

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#9129] [Questions (Nuxt 2)] [answered]
Can I use components inside my content articles??

**Comment by atinux** [comment, marked as answer]
Yes it is possible with Nuxt Content, take a look at https://content.nuxtjs.org/writing#vue-components

**Reply by JenuelDev** [reply]
oh, I tried it and it actually works, 26 days ago hehe your comment is late but, thanks a lot for replying.. Im loving nuxt, :)
